editor/src/clj/editor/camera_editor.clj

```diff
@@ -34,19 +35,22 @@
            [javax.vecmath Matrix4d Vector3d Vector4d Quat4d]))
 
 (set! *warn-on-reflection* true)
+(set! *unchecked-math* :warn-on-boxed)
```

Comment:
Turned on boxed math warnings for the file.

Boxed math happens when you have a `Double` (wrapper deriving from Object) instead of a `double` (primitive value) involved in math.

Clojure collections can store any `Object`, but primitives need to be "boxed" inside a `Double` or a `Long` wrapper in order to be stored alongside other `Objects` inside Clojure collections.

This means you'll get a boxed value back whenever you extract a primitive from a collection. Function arguments will also be boxed unless you explicitly tag them as `^double` or `^long` (however, there's a limit to only four arguments for a function with primitive arguments).

Boxed math is orders-of-magnitude slower than primitive math, so we want to avoid it if we can.

Turning on `:warn-on-boxed` will print a warning when you reload the file for each line where boxed math is happening. I've gone over the code and addressed places where boxed math was occurring.